src/IonParserTextRaw.ts

```diff
@@ -520,20 +517,20 @@ export class ParserTextRaw {
     }
   }
 
-  private _read_value_helper_single( ch1, accept_operator_symbols: boolean, calling_op ) {
-    var op;
-    if (this._peek("\'\'") != ERROR) {
-      op = this._read_string3;
-    }
-    else {
-      op = this._read_string1;
-    }
-    op.call(this);
-    if (this._test_string_as_annotation(op)) {
-      // this was an annoation, so we need to try again for the actual value
-      this._ops.unshift( calling_op ); 
+    private _read_value_helper_single( ch1, accept_operator_symbols: boolean, calling_op ) {
```

Comment:
It looks like this definition could have these types made explicit fairly easily:
```
private _read_value_helper_single( ch1: number, accept_operator_symbols: boolean, calling_op: ReadValueHelper ) : void {
```